dev_guide/deployments.adoc

```diff
@@ -396,14 +419,34 @@ no strategy is specified on a deployment configuration.
 [[rolling-strategy]]
 === Rolling Strategy
 
-The rolling strategy performs a rolling update and supports
-xref:lifecycle-hooks[lifecycle hooks] for injecting code into the deployment
-process.
+A rolling deployment slowly replaces instances of the previous version of an application
+with instances of the new version of the application. A rolling deployment typically waits
+for new pods to become *ready* via a *readiness check* before scaling down the old components.
+If a significant issue occurs, the rolling deployment can be aborted.
 
-The rolling deployment strategy waits for pods to pass their
-xref:../dev_guide/application_health.adoc#dev-guide-application-health[readiness
-check] before scaling down old components, and does not allow pods that do not
-pass their readiness check within a configurable timeout.
+=== Canary checks
```

Comment:
Canary deployments maybe... Test is a different concept and I think we need docs for them too.